documentation/api/io.strimzi.api.kafka.model.connect.ExternalConfiguration.adoc

```diff
@@ -76,14 +77,30 @@ spec:
 [id='property-kafka-connect-external-volumes-{context}']
 === `volumes`
 
-You can also mount ConfigMaps or Secrets to a Kafka Connect pod as volumes.
+Use volumes to mount ConfigMaps or Secrets to a Kafka Connect pod.
 
 Using volumes instead of environment variables is useful in the following scenarios:
 
-* Mounting truststores or keystores with TLS certificates
 * Mounting a properties file that is used to configure Kafka Connect connectors
+* Mounting truststores or keystores with TLS certificates
+
+Configuration providers load the values from mounted ConfigMaps or Secrets.
+
+Volumes are mounted inside the Kafka Connect containers in the path `/opt/kafka/external-configuration/_<volume-name>_`.
```

Comment:
_at_ the path, or _on_ the path. Or even "using the path ... as the mount point".